cloudFiles/ingress-resoure.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
The name of this file should be `ingress-resource.yaml`.

Reply:
thats the api version, not the file name